### PR TITLE
update FAQ page: added section for free dataset, python wrapper, etc.

### PR DESCRIPTION
**Pull request by qindazhu**
@danpovey could you give more comments on section TIMIT and PyTorch integration in section Python wrapper, I mean, if you think it's necessary.

### CODE REVIEW

File: src/doc/faq.dox [hunk 1]
```diff
@@ -45,6 +45,40 @@ namespace kaldi {
   - Speech Recognition with Weighted Finite-State Transducers for WSFT
   - A Bit of Progress in Language Modeling (Extended Version)
   
+  For those who may want a "Kaldi Book" with turorial on theory and implementation like what HTK Book does, we would generally just say sorry. As Dan explains in this <a href=https://groups.google.com/forum/#!topic/kaldi-help/3LBSzmploC0> post </a>, the field of speech recognition is moving so fast that we need to implement too many things in Kaldi and have no time to write such a book.
```
Comment (jtrmal) [inline src/doc/faq.dox]: turorial -> tutorial

**Comment by danpovey** [comment]
You can write in the pytorch part:
A PyTorch-integrated version of Kaldi is in the planning stage.

File: src/doc/faq.dox (hunk 1 above)
Comment (qindazhu) [reply src/doc/faq.dox]: Thanks jtrmal for the comments. I have fixed the typos. I'll check the spellings in the following PR with grammar check tool. Thanks in advance for your review.

**Comment by qindazhu** [comment]
> You can write in the pytorch part:
> A PyTorch-integrated version of Kaldi is in the planning stage.

Thanks Dan. I have updated the Python section. This PR is ready to merge if there's no other issue from your perspective.

**Comment by danpovey** [comment]
Merging.  BTW, Guoguo Chen is working on a book, but it's in Chinese.

**Comment by qindazhu** [comment]
Great! @chenguoguo if the book is ready, then I can try to translate it to an English version. If it's not so ready, maybe we can work it out together.

**Pull request merged**